### PR TITLE
Fixed applications not containing the TOS and Privacy strings.

### PR DESCRIPTION
**Pull request by FeroxFoxxo**
## Fixed Application Modal
This is a PR fixing the application modal. It adds two fields, the TOS and Privacy Policy, fields that are in Discord's API but not DNet's.

#### Related issue: https://github.com/discord-net/Discord.Net/issues/1968

### CODE REVIEW

File: src/Discord.Net.Core/Entities/IApplication.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public interface IApplication : ISnowflakeEntity
         ///     Gets the RPC origins of the application.
         /// </summary>
         IReadOnlyCollection<string> RPCOrigins { get; }
+
```
Comment (quinchs) [inline src/Discord.Net.Core/Entities/IApplication.cs]: Remove newline

File: src/Discord.Net.Rest/API/Common/Application.cs [hunk 2]
```diff
@@ -29,5 +29,10 @@ internal class Application
         [JsonProperty("owner")]
         public Optional<User> Owner { get; set; }
         public Optional<string[]> Tags { get; set; }
```
Comment (quinchs) [inline src/Discord.Net.Rest/API/Common/Application.cs:30]: Can you add a `[JsonProperty("tags")]` above this

**Review by quinchs — APPROVED** [review]

**Review by quinchs — CHANGES_REQUESTED** [review]
mean't to request changes

**Review by FeroxFoxxo — COMMENTED** [review]
Fixed all the issues given and added documentation for flags rather than removing the new line.

**Review by quinchs — APPROVED** [review]

**Pull request merged**